server/docs/game.yaml

```diff
@@ -139,3 +139,63 @@ paths:
                   value:
                     success: false
                     error: 'Player does not exist'
+
+  /game/{gameId}:
```

Comment:
the path here doesn't match what's in the router, i think the route you defined was `/game/:gameId/details`

```suggestion
  /game/{gameId}/details:
```

Reply:
Okay I fixed it!